query_panel/main.js

```diff
@@ -419,6 +472,21 @@ const app = createApp({
           this.focusPreviewPane();
           this.loading = true;
           this.timeExecution();
+          const now = Date.now();
+          this.showHint = false;
+          if (this.lastHintTimestamp + HINT_VISIBILITY_DELAY < now) {
+            this.showHint = true;
+            this.lastHintTimestamp = now;
+            HINTS.sort(() => Math.random() - 0.5);
+            executeCommand("setContext", {
```

Comment:
Should we not await setContext?

```suggestion
            await executeCommand("setContext", {
```

Reply:
no. later actions doesn't depend on its result. And assuming it will successfully complete.